static/js/apps/explore/app.tsx

```diff
@@ -213,6 +213,7 @@ export function App(): JSX.Element {
   const cmpPlace = getSingleParam(hashParams["pcmp"]);
   const topic = getSingleParam(hashParams["t"]);
   const placeType = getSingleParam(hashParams["pt"]);
+  const sdg = getSingleParam(hashParams["sdg"]);
```

Comment:
nit: Should we make this `dc=sdg` or `config=sdg` or similar?

Reply:
changed to dc=sdg